src/components/Badge/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import { BaseProps } from '../types';
+import { StyledBadge, StyledBadgeContainer } from './styled';
+
+type badgeVariant = 'default' | 'lightest' | 'outline-brand' | 'brand';
+
+interface Props extends BaseProps {
+    label?: ReactNode;
+    variant?: badgeVariant;
+    style?: object;
+}
+
+/**
+ * Badges are labels which hold small amounts of information.
+ */
+
+const Badge: React.FC<Props> = props => {
+    const { label, variant, style } = props;
+    return (
+        <StyledBadgeContainer style={style}>
+            <StyledBadge variant={variant}>{label}</StyledBadge>
+        </StyledBadgeContainer>
+    );
+};
+
+Badge.propTypes = {
+    label: PropTypes.node,
+    variant: PropTypes.oneOf(['default', 'lightest', 'outline-brand', 'brand']),
+    style: PropTypes.object,
```

Comment:
we need to add the description added in the component spec:
https://docs.google.com/document/d/1uAlNEjAljHL3aj-824LaFOEI0EeYl8L6ungCjPfW5Uk/edit
something like:
```
Badge.propTypes = {
     /** The text to be displayed inside the badge. */
    label: PropTypes.node,
    /** The variant changes the appearance of the badge.
     * Accepted variants include default, lightest, brand and outline-brand.
     * This value defaults to default. */
    variant: PropTypes.oneOf(['default', 'lightest', 'outline-brand', 'brand']),
    /** An object with custom style applied to the outer element. */
    style: PropTypes.object,
};
```
as we did with our other components in
https://github.com/nexxtway/react-rainbow